main/config/settings/.defaults.yml

```diff
@@ -179,3 +179,5 @@ versionDate: '2021-09-16'
 
 # Metadata provided with AWS SDK calls
 customUserAgent: 'AwsLabs/SO0144/${self:custom.settings.versionNumber}'
+
```

Comment:
Could we point to a URL or include something in the example.yml file to show a list of all available tags in the open data registry?

Reply:
Yep, I'll add that in the comment. The list of available tags is [here](https://github.com/awslabs/open-data-registry/blob/main/tags.yaml)